config.md

```diff
@@ -91,7 +91,7 @@
 |---|-----------|----|-------------|
 |count|The maximum number of times to retry|`int`|`5`
 |enabled|Enables retries|`boolean`|`false`
-|factor|The retry backoff factor|`boolean`|`2`
+|factor|The retry backoff factor|`int`|`2`
```

Comment:
I'm actually confused by this one. This should be picking up the global config, all the way up from here - but that doesn't seem to have the bug 🤔 
https://github.com/hyperledger/firefly-common/blob/5a2539a3727b6eeecee07da0297198f7bcbd2bed/pkg/i18n/en_base_config_descriptions.go#L44

Reply:
[Here's the culprit](https://github.com/hyperledger/firefly-common/blob/5a2539a3727b6eeecee07da0297198f7bcbd2bed/pkg/i18n/en_base_config_descriptions.go#L30)

```
var FloatType = "`boolean`"
```

Reply:
🤦

Reply:
https://github.com/hyperledger/firefly-common/pull/36